include/vara/Feature/Constraint.h

```diff
@@ -19,7 +19,7 @@ namespace vara::feature {
 
 class ConstraintVisitor;
 
-/// \brief Tree like representation of constraints between features
+/// \brief Tree like representation of constraints between features3
```

Comment:
```suggestion
/// \brief Tree like representation of constraints between features
```
I assume the `3` does not carry meaning :wink: